functions/src/modules/SE.functions.ts

```diff
@@ -16,15 +16,13 @@ export const uploadProcessing = functions.storage.object()
 
     const filePath = object.name;
     
-    if (object.name.slice(-4).toLowerCase() !== 'flac') {
-        console.error("Something is wrong with the file extension!")
-        return -1;
-    }
-
     const filePathRegex = /sound-editing\/restored\/(\w+)\/(\w+)/g;
 
-    if (!filePath.match(filePathRegex)) {
-        console.error("Something is wrong with the file path!")
+    if (!filePath.match(filePathRegex)) 
+        return -1;    
+
+    if (object.contentType === 'audio/flac') {
+        console.error("Something is wrong with the file extension!")
```

Comment:
The message should mention the file type, not an extension. The file can have FLAC content type and another extension.